.sequelizerc

```diff
@@ -0,0 +1,9 @@
+
+const path = require('path');
+
+module.exports = {
+  'config': path.resolve('./src/db/config', 'babelHook.js'),
+  'models-path': path.resolve('src/db', 'models'),
+  'seeders-path': path.resolve('src/db', 'seeders'),
```

Comment:
Seeders and Migrations can be in the config folder outside the src folder. This is to avoid us transpiling code we do not need.
```suggestion
  'seeders-path': path.resolve('./config', 'seeders'),
```